mango-python/src/mango/QC.py

```diff
@@ -31,28 +31,47 @@ def __init__(self):
         Initializes a Quality Control class.
         """
 
-    def CoverageDistribution(self, coverageRDD, showPlot = True):
-
-        # make sure coverageRDD is flattened
-        flattened = coverageRDD.flatten()
-
-        # compute coverage distribution
-        coverageDistribution = flattened.toDF().rdd \
+    def CoverageDistribution(self, coverageRDD, showPlot=True, normalize = False, cummulative = False,):
```

Comment:
maybe we can do some sequence(CoverageRDDs) instead of CoverageRDD so we can create a plot on multiple files